<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QMainWindow{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.201, fx:0.494, fy:0.505864, stop:0 rgba(98, 98, 98, 255), stop:1 rgba(255, 255, 255, 255))
}
QPushButton
{
color:white;
background-color:Black;
border: 2px solid white;
border-radius: 10px;
padding: 10px;
margin: 5px;
}

QPushButton:hover
{
background-color:white;
color:red;
}


QTextEdit
{
color:white;
background-color:Black;
border: 2px solid white;
padding: 10px;
margin: 5px;
}

QcomboBox
{
color: black;
background-color:Black;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_5">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <family>Stencil</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="5">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Showcard Gothic</family>
            <pointsize>12</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>GAG-CCENT GENERATOR</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="5">
         <widget class="QTextEdit" name="textEdit">
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="5">
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>Stencil</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Send it!</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="5">
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <family>Terminal</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>OpenHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Choose an accent!</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medieval</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>French</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pirate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Backwards</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spoiled Victorian Child</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Binary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Brainrot</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QLabel" name="logo">
          <property name="styleSheet">
           <string notr="true">   Image
    {
        id:underlyingImage
        width: 400
        height: 400
        x:0;
        y:0;
        source:&quot;pngimg.com - crusader_PNG42.png&quot;
        z:1
    }</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QTextBrowser" name="result_box">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ForbiddenCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="AgainBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QLabel" name="AgainBox2">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="4">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Small Fonts</family>
           </font>
          </property>
          <property name="text">
           <string>                                                                           Input your text here!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>150</y>
          <width>551</width>
          <height>121</height>
         </rect>
        </property>
        <property name="baseSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>60</pointsize>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>GAG-CCENT GENERATOR!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>260</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Start!</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="funLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="FFLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
